CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/AnalyticConfigurationPane.java

```diff
@@ -582,4 +620,71 @@ public String getName() {
             return "Analytic View: Update State";
         }
     }
+    
+    /**
+     * Update the display by reading and writing to/from the state attribute. 
+     */
+    private static final class AnalyticViewStateUpdater extends SimpleEditPlugin {
+
+        private final AnalyticConfigurationPane analyticConfigurationPane;
+        private final boolean pluginWasSelected;
+        
+        public AnalyticViewStateUpdater(final AnalyticConfigurationPane analyticConfigurationPane, final boolean categoryWasSelected) {
+            this.analyticConfigurationPane = analyticConfigurationPane;
+            this.pluginWasSelected = categoryWasSelected;
```

Comment:
Should this be categoryWasSelected?

Reply:
I did some refactoring of names to make it more readable, but forgot to change that. I'll fix that.

Reply:
A missed variable when refactoring for ease of reading. Ill get that fixed.